bin/qi

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
 
-set -e
-
-cd "$(dirname ${BASH_SOURCE[0]})/../"
+QI_PREFIX=${QI_PREFIX:-"$(dirname ${BASH_SOURCE[0]})/.."}
```

Comment:
This is for packagers that might symlink the binary elsewhere; they can make a wrapper script that sets this variable to wherever the Qi source is.